docs/components/NetworkContext.tsx

```diff
@@ -0,0 +1,45 @@
+import { createContext, useEffect, useState } from "react";
+import { Network } from "@glow-app/glow-client";
+
+const NETWORK_LOCAL_STORAGE_KEY = "nftoken-docs-network";
+type Context = {
+  network: Network;
+  setNetwork: (network: Network) => void;
+  networkPrettyName: string;
```

Comment:
This feels weird. Let's instead have an object `const NETWORK_TO_INFO: Record<Network, {name: string}>`. We usually want to keep state as small as possible.